src/plugins/data-card-tool/data-card-tool.scss

```diff
@@ -209,6 +209,14 @@ $default-button-border-color: #979797;
         }
       }
 
+      .delete-attribute {
+        display:none;
+        transform: translateX(30px);
```

Comment:
👍 This seems like a better approach than the margins I used to accomplish a similar purpose.

Reply:
I actually ended up changing this.  Above, `position:relative` made the input shrink to make room for the spot the button would have been had it not been translated.  So I ended up with `position:absolute`. 

```scss
.delete-attribute {
     position: absolute;
     display:none;
     transform: translateX(351px);
}
```